src/Database/InMemoryProvider.js

```diff
@@ -61,19 +65,18 @@ class InMemoryProvider extends ADBProvider {
     }
 
     saveAxonSchema(axonSchema) {
```

Comment:
Apparently and this will the last change needed in this PR, these methods are both named wrongly
```suggestion
    saveAxon(axonSchema) {
```